packages/core/src/scripts/tests/migrations.test.ts

```diff
@@ -405,35 +405,35 @@ describe('useMigrations: true', () => {
     `);
 
     expect(cleanOutputForApplyingMigration(recording(), migrationName)).toMatchInlineSnapshot(`
-      "✨ Starting Keystone
-      ⭐️ Server listening on :3000 (http://localhost:3000/)
-      ⭐️ GraphQL API available at /api/graphql
-      ✨ Generating GraphQL and Prisma schemas
-      ✨ There has been a change to your Keystone schema that requires a migration
+      "? Starting Keystone
+      ? Server listening on :3000 (http://localhost:3000/)
+      ? GraphQL API available at /api/graphql
+      ? Generating GraphQL and Prisma schemas
+      ? There has been a change to your Keystone schema that requires a migration
 
-      ⚠️  Warnings:
+      ?  Warnings:
 
-        • You are about to drop the column \`title\` on the \`Todo\` table, which still contains 1 non-null values.
+        ? You are about to drop the column \`title\` on the \`Todo\` table, which still contains 1 non-null values.
 
       Prompt: Name of migration remove all fields except id
-      ✨ A migration has been created at migrations/migration_name
+      ? A migration has been created at migrations/migration_name
       Prompt: Would you like to apply this migration? true
       Applying migration \`migration_name\`
-      ✨ Connecting to the database
-      ✨ Creating server
-      ✅ GraphQL API ready"
+      ? Connecting to the database
+      ? Creating server
+      ? GraphQL API ready"
     `);
   });
   test('prompts to drop database when database is out of sync with migrations directory', async () => {
     const { migrationName: oldMigrationName, prevCwd } = await setupInitialProjectWithMigrations();
     const tmp = await testdir({
       ...symlinkKeystoneDeps,
       'app.db': await fs.readFile(`${prevCwd}/app.db`),
-      'keystone.js': await fs.readFile(`${__dirname}/fixtures/one-field-with-migrations.ts`),
+      'keystone.js': await fs.readFile(`${__dirname}/fixtures/one-field-with-migrations-2.ts`),
     });
     const recording = recordConsole({
       'Do you want to continue? All data will be lost.': true,
-      'Name of migration': 'init',
```

Comment:
```
{
  "oldMigrationName": "20230504081137_init",
  "migrationName": "20230504081138_init2"
}
```

This was previously failing when the timestamps were the same, and hence the migration names were the same